cmd/kubelet/app/server.go

```diff
@@ -502,6 +502,20 @@ func InitializeTLS(kc *componentconfig.KubeletConfiguration) (*server.TLSOptions
 		CertFile: kc.TLSCertFile,
 		KeyFile:  kc.TLSPrivateKeyFile,
 	}
+
+	if ks.TLSCertAuth {
```

Comment:
ideally this would look like the master's TLS config options: https://github.com/kubernetes/kubernetes/blob/master/pkg/genericapiserver/genericapiserver.go#L705-L715